build.rs

```diff
@@ -20,6 +20,152 @@ fn build_config_name() -> String {
     format!("mkl-dynamic-{}-{}", integer_config, parallelism)
 }
 
+/// Paths required for linking to MKL from MKLROOT folder
+struct MklDirectories {
+    #[allow(dead_code)]
+    mkl_root: String,
+    lib_dir: String,
+    omp_lib_dir: String,
+    include_dir: String,
```

Comment:
Couldn't we also use e.g. `PathBuf` here, since they are paths...? Or is that less convenient somehow...?